### PR TITLE
Create verify-signed-images.md

### PR DESCRIPTION
**Pull request by ihuibin**
/assign  @tengqm
Signed-off-by: bin.hui <bin.hui@daocloud.io>

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 1]
```diff
@@ -0,0 +1,129 @@
+---
+title: 验证已签名容器镜像
+content_type: task
+min-kubernetes-server-version: v1.24
+---
+<!--
+---
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:7]: remove this line

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+title: Verify Signed Container Images
+content_type: task
+min-kubernetes-server-version: v1.24
+---
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:11]: remove this line

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 3]
```diff
@@ -0,0 +1,129 @@
...
+COSIGN_EXPERIMENTAL=1 cosign verify k8s.gcr.io/kube-apiserver-amd64:v1.24.0
+```
+
+{{< note >}}
+`COSIGN_EXPERIMENTAL=1` is used to allow verification of images signed
+in `KEYLESS` mode. To learn more about keyless signing, please refer to
+[Keyless Signatures](https://github.com/sigstore/cosign/blob/main/KEYLESS.md#keyless-signatures)
+. {{< /note >}}
+-->
+
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:53]: remove this line

File: content/zh/docs/tasks/administer-cluster/verify-signed-images.md [hunk 4]
```diff
@@ -0,0 +1,129 @@
...
+```
+
+{{< note >}}
+`COSIGN_EXPERIMENTAL=1` 用于对以 `KEYLESS` 模式签名的镜像进行验证。想要进一步了解 `KEYLESS`，请参考
+[Keyless Signatures](https://github.com/sigstore/cosign/blob/main/KEYLESS.md#keyless-signatures)。
+{{< /note >}}
+<!--
+### Verifying images for all control plane components
+
+To verify all signed control plane images, please run this command:
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/verify-signed-images.md:71]: 请先翻译69-71行。
之后保留 72-80 行不动。
再翻译82-88 行.

这种大段地翻译需要前后翻屏地 review，很难对比中英文，并且代码段根本不需要翻译（其中不含注释），注释一份再粘贴一份毫无意义。

**Comment by tengqm** [comment]
This one is related to #33477.

**Comment by ihuibin** [comment]
好的，我现在应该咋做？ @Sea-n

**Comment by Sea-n** [comment]
You should squash your commits into one and modify the commit message. I would recommend you to use GitHub Desktop since it have a friendly GUI.

你該透過本機的 git 客戶端將所有 commit squash 成一個，建議可以安裝 [GitHub Desktop](https://desktop.github.com/) 並參考 [這篇官方文件](https://docs.github.com/cn/desktop/contributing-and-collaborating-using-github-desktop/managing-commits/squashing-commits)

在 squash 成 1 筆 commit 後，需要編輯 commit 訊息並拿掉 `/assign @...` 等關鍵字

完成並 force-push 上來後， @k8s-ci-robot 就會把「不可合併/無效的 commit 訊息」標籤移除了

**Comment by ydFu** [comment]
Thank you @Sea-n  for the suggestion.

Thank you @ihuibin  for your contribution to `kubernetes/website` and give the following tips
 1. Use [`git rebase`](https://git-scm.com/docs/git-rebase) to squash/fixup dummy/unnecessary commits into only one commit. 
 2. It is strictly **forbidden to use reserved words** (e.g. `/assign @...`, `fix #xxx`) in the git commit message, which will cause **k8s-ci-robot** CI to judge an illegal PR, and add the `do-not-merge/invalid-commit-message` label. This will result in the inability to continue code review.

**Comment by Sea-n** [comment]
Hey @ihuibin, we have moved Mandarin contain from `content/zh/docs` to `content/zh-cn/docs`, please rebase and move you file accordingly.

You can `/unhold` by yourself once it's done.
/hold

**Comment by tengqm** [comment]
@ihuibin This one has been stagnated for too long. Are you still working on it?

**Comment by ihuibin** [comment]
YES,I working on it , but I do not know  how remove [do-not-merge/invalid-commit-message] tag.

**Comment by tengqm** [comment]
> YES,I working on it , but I do not know how remove [do-not-merge/invalid-commit-message] tag.

In you git commit message, do not mention anybody or any issue.
In your PR message, you can mention the issue to close or the issue related.

**Pull request closed without merging**